packages/chakra-ui-core/src/CAvatarGroup/CAvatarGroup.js

```diff
@@ -40,25 +41,27 @@ const CMoreAvatarLabel = {
   render (h) {
     const borderColor = { light: '#fff', dark: 'gray.800' }
     const bg = { light: 'gray.200', dark: 'whiteAlpha.400' }
-
-    const theme = this.theme
     const sizeKey = avatarSizes[this.size]
-    const _size = theme.sizes[sizeKey]
+    const _size = this.theme.sizes[sizeKey]
     const fontSize = `calc(${_size} / 2.75)`
 
     return h(CFlex, {
       props: {
+        align: 'center',
+        justify: 'center',
+        ...forwardProps(this.$props)
+      },
+      attrs: {
         w: avatarSizes[this.size],
         h: avatarSizes[this.size],
         bg: bg[this.colorMode],
         color: 'inherit',
         rounded: 'full',
-        alignItems: 'center',
-        justifyContent: 'center',
         border: '2px',
         borderColor: borderColor[this.colorMode],
         fontSize,
-        ...forwardProps(this.$props)
+        ...this.$attrs,
+        'data-chakra-component': 'CMoreAvatarLabel'
```

Comment:
why this is not a functional component? or why not use `createStyledAttrsMixin` ?

Reply:
I made this component earlier on. Need to see why I didn't make it one of the two.

Reply:
Good catch. These are now both functional components.